logpolicy/logpolicy.go

```diff
@@ -782,6 +824,7 @@ func NewLogtailTransport(host string, netMon *netmon.Monitor, logf logger.Logf)
 		tr.TLSNextProto = map[string]func(authority string, c *tls.Conn) http.RoundTripper{}
 	}
 
+	host := cmp.Or(opts.Host, logtail.DefaultHost)
```

Comment:
is this new behavior?

Reply:
Technically, yes. Practically, no.

All callers were already passing in `logtail.DfaultHost`, so no change in behavior there.

On observable change is if you specify the undocumented TS_LOG_TARGET environment variable and put an invalid URL in there (such that the parsed host is empty).